google-cloud-spanner/src/test/java/com/google/cloud/spanner/SpannerOptionsThreadTest.java

```diff
@@ -54,12 +52,9 @@ private SpannerOptions createOptions() {
         .setProjectId("p")
         // Set a custom channel configurator to allow http instead of https.
         .setChannelConfigurator(
-            new ApiFunction<ManagedChannelBuilder, ManagedChannelBuilder>() {
-              @Override
-              public ManagedChannelBuilder apply(ManagedChannelBuilder input) {
-                input.usePlaintext();
-                return input;
-              }
+            input -> {
+              input.usePlaintext();
```

Comment:
nit: can be further simplfied to `input -> input.usePlaintext()`

Reply:
Fixed.